src/features/channels/components/chat-header/ChatHeader.tsx

```diff
@@ -55,9 +54,9 @@ const ChatHeader = () => {
             <MoreHorizIcon />
           </IconButton>
         </Box>
-        <Box className={classes.account}>
+        {/* <Box className={classes.account}>
```

Comment:
Same than above, please remove commented blocks of code

Reply:
ok, I will remove the comments